design/move-gh-org.md

```diff
@@ -0,0 +1,80 @@
+# Plan for moving the Velero GitHub repo into the VMware GitHub organization
+
+Currently, the Velero repository sits under the Heptio GitHub organization. With the acquisition of Heptio by VMware, it is due time that this repo moves to one of the VMware GitHub organizations. This document outlines a plan to move this repo to the VMware Tanzu (https://github.com/vmware-tanzu) organization.
+
+## Goals
+
+- List all steps necessary to have this repo fully functional under the new org
+
+## Non Goals
+
+- Highlight any step necessary around setting up the new organization and its members
+
+## Action items
+
+### Todo list
```

Comment:
Thanks for pulling this list together. It'd be helpful to see it in order by timeline - e.g. first the items that we can prep before the repo move (e.g. find/replace PRs, write the blog post, etc), then the repo move itself, then all the things that need to be set up post-move (GH integrations, merging PRs, etc).